leader-election/DistributedMutex/BlobLeaseManager.cs

```diff
@@ -88,6 +86,12 @@ public async Task<string> AcquireLeaseAsync(CancellationToken token)
                     return null;
                 }
             }
+            catch (Exception e)
+            {
+                // If the storage account is unavailable or we fail for any other reason we still want to keep retrying
+                Trace.TraceError(e.Message);
+                return null;
+            }
```

Comment:
You asked about this in the teams chat.  I'm okay with this in here.  I think it at least shows that "SOMETHING" needs to happen here, and this is a reasonable something for sure.